subprojects/docs/src/docs/userguide/running-builds/build_environment.adoc

```diff
@@ -53,6 +53,21 @@ _By default, the build cache is *not* enabled._
 When set to true, individual input property hashes and the build cache key for each task are logged on the console. Learn more about <<build_cache.adoc#sec:task_output_caching, task output caching>>.
 _Default is `false`._
 
+`org.gradle.configuration-cache=(true,false)`::
```

Comment:
Is that change needed?

Reply:
@abstratt I have your commit tagged along in my PR, may you please check it and confirm we are good to merge it altogether?

Reply:
@kate0 Sorry, only saw this now. I must admit I don't understand how my commit ended up in your branch.